docs/compatibility.md

```diff
@@ -235,27 +235,51 @@ from one run to another if the ordering is ambiguous on a window function too.
 ## Parsing strings as dates or timestamps
 
 When converting strings to dates or timestamps using functions like `to_date` and `unix_timestamp`,
-only a subset of possible formats are supported on GPU with full compatibility with Spark. The
-supported formats are:
+the specified format string will fall into one of three categories:
+
+- Supported on GPU, with compatibility with Spark
+- Supported on GPU, but may produce different results to Spark
+- Unsupported on GPU
+
+The supported formats are:
```

Comment:
Which version of "supported" are these?  The above bullets list two forms of supported, and it's ambiguous whether these are just the 100% supported or both forms of supported.